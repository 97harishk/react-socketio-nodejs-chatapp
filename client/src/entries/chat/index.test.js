import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import {
	HomeEntry
} from 'entries';

describe('<HomeEntry />', () => {
	const modelI = {};

	const getWrapper = (model) => {
		return shallow((
			<HomeEntry {...model} />
		));
	};

	it('renders correctly', () => {
		const wrapper = getWrapper(modelI);
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
