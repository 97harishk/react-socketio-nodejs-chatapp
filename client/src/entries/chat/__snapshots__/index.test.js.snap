// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChatEntry /> renders correctly 1`] = `
<div
  className="chat-wrapper"
>
  <div
    className="actions-container"
  >
    <header
      className="header-container"
    >
      <div
        className="header-content"
      >
        <div>
          <ProfilePictureComponent
            backgroundColor="#1461ff"
            color="white"
            height={60}
            label="AM"
            width={60}
          />
        </div>
        <div>
          <ButtonComponent
            height={26}
            link={true}
            onClick={[Function]}
            type="button"
            width={26}
          >
            <IconComponent
              fill="#555657"
              height={26}
              icon="account-plus"
              width={26}
            />
          </ButtonComponent>
          <ButtonComponent
            height={26}
            link={true}
            margin="0px 0px 0px 20px"
            onClick={[Function]}
            type="button"
            width={26}
          >
            <IconComponent
              fill="#555657"
              height={26}
              icon="message-text"
              width={26}
            />
          </ButtonComponent>
        </div>
      </div>
      <LabelComponent
        dark={true}
        fontSemiBold={true}
        fontSize={16}
        margin="10px 0px 0px 0px"
        text="Amendowins"
      />
    </header>
    <div>
      <InputComponent
        autoComplete="off"
        iconComponent={[Function]}
        maxLength={15}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        searchButton={true}
        type="text"
      />
    </div>
    <div
      className="conversations-container"
    >
      <div
        key="0"
      >
        <div
          className="card"
        >
          <UserCardContainer
            desc={
              Object {
                "fontSize": 13,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "Hey there! how are you",
              }
            }
            profile={
              Object {
                "backgroundColor": "#1461ff",
                "color": "white",
                "height": 40,
                "label": "AM",
                "labelFontSize": 12,
                "width": 40,
              }
            }
            title={
              Object {
                "fontSize": 14,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "123456789123456",
              }
            }
          />
          <LabelComponent
            fontRegular={true}
            fontSize={13}
            margin="0px 0px 0px 0px"
            text="Yesterday"
          />
        </div>
        <span
          className="card-division"
        />
      </div>
      <div
        key="1"
      >
        <div
          className="card"
        >
          <UserCardContainer
            desc={
              Object {
                "fontSize": 13,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "Hey there! how are you",
              }
            }
            profile={
              Object {
                "backgroundColor": "#dbcc86",
                "color": "white",
                "height": 40,
                "label": "AM",
                "labelFontSize": 12,
                "width": 40,
              }
            }
            title={
              Object {
                "fontSize": 14,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "123456789123456",
              }
            }
          />
          <LabelComponent
            fontRegular={true}
            fontSize={13}
            margin="0px 0px 0px 0px"
            text="Yesterday"
          />
        </div>
        <span
          className="card-division"
        />
      </div>
      <div
        key="2"
      >
        <div
          className="card"
        >
          <UserCardContainer
            desc={
              Object {
                "fontSize": 13,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "Hey there! how are you",
              }
            }
            profile={
              Object {
                "backgroundColor": "#d89389",
                "color": "white",
                "height": 40,
                "label": "AM",
                "labelFontSize": 12,
                "width": 40,
              }
            }
            title={
              Object {
                "fontSize": 14,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "123456789123456",
              }
            }
          />
          <LabelComponent
            fontRegular={true}
            fontSize={13}
            margin="0px 0px 0px 0px"
            text="Yesterday"
          />
        </div>
        <span
          className="card-division"
        />
      </div>
      <div
        key="3"
      >
        <div
          className="card"
        >
          <UserCardContainer
            desc={
              Object {
                "fontSize": 13,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "Hey there! how are you",
              }
            }
            profile={
              Object {
                "backgroundColor": "#1461ff",
                "color": "white",
                "height": 40,
                "label": "AM",
                "labelFontSize": 12,
                "width": 40,
              }
            }
            title={
              Object {
                "fontSize": 14,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "123456789123456",
              }
            }
          />
          <LabelComponent
            fontRegular={true}
            fontSize={13}
            margin="0px 0px 0px 0px"
            text="Yesterday"
          />
        </div>
        <span
          className="card-division"
        />
      </div>
      <div
        key="4"
      >
        <div
          className="card"
        >
          <UserCardContainer
            desc={
              Object {
                "fontSize": 13,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "Hey there! how are you",
              }
            }
            profile={
              Object {
                "backgroundColor": "#dbcc86",
                "color": "white",
                "height": 40,
                "label": "AM",
                "labelFontSize": 12,
                "width": 40,
              }
            }
            title={
              Object {
                "fontSize": 14,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "123456789123456",
              }
            }
          />
          <LabelComponent
            fontRegular={true}
            fontSize={13}
            margin="0px 0px 0px 0px"
            text="Yesterday"
          />
        </div>
        <span
          className="card-division"
        />
      </div>
      <div
        key="5"
      >
        <div
          className="card"
        >
          <UserCardContainer
            desc={
              Object {
                "fontSize": 13,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "Hey there! how are you",
              }
            }
            profile={
              Object {
                "backgroundColor": "#d89389",
                "color": "white",
                "height": 40,
                "label": "AM",
                "labelFontSize": 12,
                "width": 40,
              }
            }
            title={
              Object {
                "fontSize": 14,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "123456789123456",
              }
            }
          />
          <LabelComponent
            fontRegular={true}
            fontSize={13}
            margin="0px 0px 0px 0px"
            text="Yesterday"
          />
        </div>
        <span
          className="card-division"
        />
      </div>
      <div
        key="6"
      >
        <div
          className="card"
        >
          <UserCardContainer
            desc={
              Object {
                "fontSize": 13,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "Hey there! how are you",
              }
            }
            profile={
              Object {
                "backgroundColor": "#1461ff",
                "color": "white",
                "height": 40,
                "label": "AM",
                "labelFontSize": 12,
                "width": 40,
              }
            }
            title={
              Object {
                "fontSize": 14,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "123456789123456",
              }
            }
          />
          <LabelComponent
            fontRegular={true}
            fontSize={13}
            margin="0px 0px 0px 0px"
            text="Yesterday"
          />
        </div>
        <span
          className="card-division"
        />
      </div>
      <div
        key="7"
      >
        <div
          className="card"
        >
          <UserCardContainer
            desc={
              Object {
                "fontSize": 13,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "Hey there! how are you",
              }
            }
            profile={
              Object {
                "backgroundColor": "#dbcc86",
                "color": "white",
                "height": 40,
                "label": "AM",
                "labelFontSize": 12,
                "width": 40,
              }
            }
            title={
              Object {
                "fontSize": 14,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "123456789123456",
              }
            }
          />
          <LabelComponent
            fontRegular={true}
            fontSize={13}
            margin="0px 0px 0px 0px"
            text="Yesterday"
          />
        </div>
        <span
          className="card-division"
        />
      </div>
      <div
        key="8"
      >
        <div
          className="card"
        >
          <UserCardContainer
            desc={
              Object {
                "fontSize": 13,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "Hey there! how are you",
              }
            }
            profile={
              Object {
                "backgroundColor": "#d89389",
                "color": "white",
                "height": 40,
                "label": "AM",
                "labelFontSize": 12,
                "width": 40,
              }
            }
            title={
              Object {
                "fontSize": 14,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "123456789123456",
              }
            }
          />
          <LabelComponent
            fontRegular={true}
            fontSize={13}
            margin="0px 0px 0px 0px"
            text="Yesterday"
          />
        </div>
        <span
          className="card-division"
        />
      </div>
      <div
        key="9"
      >
        <div
          className="card"
        >
          <UserCardContainer
            desc={
              Object {
                "fontSize": 13,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "Hey there! how are you",
              }
            }
            profile={
              Object {
                "backgroundColor": "#1461ff",
                "color": "white",
                "height": 40,
                "label": "AM",
                "labelFontSize": 12,
                "width": 40,
              }
            }
            title={
              Object {
                "fontSize": 14,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "123456789123456",
              }
            }
          />
          <LabelComponent
            fontRegular={true}
            fontSize={13}
            margin="0px 0px 0px 0px"
            text="Yesterday"
          />
        </div>
        <span
          className="card-division"
        />
      </div>
      <div
        key="10"
      >
        <div
          className="card"
        >
          <UserCardContainer
            desc={
              Object {
                "fontSize": 13,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "Hey there! how are you",
              }
            }
            profile={
              Object {
                "backgroundColor": "#dbcc86",
                "color": "white",
                "height": 40,
                "label": "AM",
                "labelFontSize": 12,
                "width": 40,
              }
            }
            title={
              Object {
                "fontSize": 14,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "123456789123456",
              }
            }
          />
          <LabelComponent
            fontRegular={true}
            fontSize={13}
            margin="0px 0px 0px 0px"
            text="Yesterday"
          />
        </div>
        <span
          className="card-division"
        />
      </div>
      <div
        key="11"
      >
        <div
          className="card"
        >
          <UserCardContainer
            desc={
              Object {
                "fontSize": 13,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "Hey there! how are you",
              }
            }
            profile={
              Object {
                "backgroundColor": "#d89389",
                "color": "white",
                "height": 40,
                "label": "AM",
                "labelFontSize": 12,
                "width": 40,
              }
            }
            title={
              Object {
                "fontSize": 14,
                "margin": "0px 0px 0px 14px",
                "maxWidth": 100,
                "text": "123456789123456",
              }
            }
          />
          <LabelComponent
            fontRegular={true}
            fontSize={13}
            margin="0px 0px 0px 0px"
            text="Yesterday"
          />
        </div>
        <span
          className="card-division"
        />
      </div>
    </div>
  </div>
  <div
    className="chat-container"
  >
    <header
      className="header-container"
    />
    <section
      className="chat-content"
    />
    <footer
      className="footer-container"
    />
  </div>
</div>
`;
